test/evergreen.yml

```diff
@@ -1990,6 +1990,48 @@ tasks:
           # No of processes to run in the background
           no_of_procs: 10
 
+  # The task name is ppc-zseries because this task will be used in both buildVariants
+  - name: format-stress-ppc-zseries-test
+    # Set 2.5 hours timeout (60 * 60 * 2.5)
+    exec_timeout_secs: 9000
+    commands:
+      - func: "get project"
+      - func: "compile wiredtiger with builtins"
+      - func: "format test script"
+        vars:
+          format_test_setting: ulimit -c unlimited
+          #run for 2 hours ( 2 * 60 = 120 minutes), use default config
+          format_test_script_args: -b "SEGFAULT_SIGNALS=all catchsegv ./t" -t 120
+
+  - name: format-wtperf-test
+    commands:
+      - func: "get project"
+      - func: "compile wiredtiger with builtins"
+      - command: shell.exec
+        params:
+          working_dir: "wiredtiger/build_posix"
+          script: |
+            set -o errexit
+            set -o verbose
+
+            cat > split_heavy.wtperf << EOF
+                conn_config="cache_size=100MB,log=(enabled=false),statistics=[fast],statistics_log=(wait=5,json=false)"
+                table_config="leaf_page_max=8k,internal_page_max=8k,leaf_item_max=1433,internal_item_max=3100,type=file,memory_page_max=1MB,split_deepen_min_child=100"
+                table_count=3
+                icount=50000
+                populate_threads=1
+                random_range=100000000
+                checkpoint_threads=0
+                report_interval=5
+                run_time=10
+                threads=((count=20,inserts=1))
+                value_sz=200
+                key_sz=64
+                reopen_connection=false
+            EOF
+
+            ./bench/wtperf/wtperf -O ./split_heavy.wtperf -o verbose=2
```

Comment:
It would be neater to put this wtperf configuration into a file instead of crafting it every time running the test. Could the working_dir be adjusted to `wiredtiger/build_posix/bench/wtperf`?

Reply:
Yes, Thanks for pointing it out. Fixed.